app/controllers/api/v1/api_controller.rb

```diff
@@ -49,10 +49,10 @@ def page
       @page.zero? ? 1 : @page
     end
 
-    # max limit is 25
+    # max limit is 50, default is 25
     def per_page
       @per_page = params['per_page'].to_i
-      return DEFAULT_SEARCH_COUNT if @per_page.zero?
+      return DEFAULT_SEARCH_COUNT if @per_page.zero? || @per_page.negative?
```

Comment:
or `@per_page < 1` ?!

Reply:
in case of `?per_page=-123` , not really needed but I'm filtering out all invalid values

Reply:
invalid values -> use default 
large values -> use max